src/main/java/net/pms/image/ImagesUtil.java

```diff
@@ -1479,7 +1479,7 @@ protected static Image transcodeImage(
 		Thumbnails.of(bufferedImage)
 			.scale(1.0d)
 			.outputFormat(outputFormat.toString())
-			.outputQuality(1.0f)
+			.outputQuality(0.6f)
```

Comment:
As I already said in #1527 this is effectively a bug.

Reply:
Is this better? https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1528/commits/c95f6b3bcc65dee219f526426fbb2657ccc96a84